_posts/2021-10-04-sharing-materials-from-ncwit-talk.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Sharing resources from recent NCWIT talk on counteracting bias
+tags: [events, DEI, working groups]
+posted_by: David Nicholson
+---
+
+On Thursday, August 12th, US-RSE held the
+[second talk of our speaker series](https://us-rse.org/events/2021/2021-07-dei-speaker-series-ncwit)
+organized by the Diversity, Equity, and Inclusion (DEI) working group.
+We were joined by [Joanne Esch](https://ncwit.org/profile/joanne-esch/)
+and [Andrea Bowen-Jones](https://ncwit.org/profile/andrea-bowens-jones/)
+from [The National Center for Women & Information Technology (NCWIT)](https://ncwit.org/).
+They spoke with us the biases and barriers that
+face women and other underrepresented groups
+participating in the technical workforce,
+and interventions that NCWIT has developed
+to help address these biases.
+
+We would like to pass along some of the resources that the speakers shared.
+The first resource is the NCWIT's
+[critical listening guide](https://ncwit.org/resources/critical-listening-guide/).
+This guide addresses common misunderstandings that arise during discussions of
+these topics.
+In it you'll find concrete responses and links to primary research
+to help you deal with problematic statements, like "Women need to learn to be
+more confident." The guide provides questions to ask and comments to add
+that can prevent these problematic statements from derailing a discussion.
+Two more guides we'd like to point are aimed specifically at interrupting bias
```

Comment:
```suggestion
Two more guides we'd like to highlight are aimed specifically at interrupting bias
```
or point out or whatever

Reply:
"interrupting bias" is an interesting verb for it but I surmise that it has currency in the community. Naively I'd have imagined that it's the instances of bias that are to be interrupted. Still, not saying to change anything, just thinking out loud.

Reply:
"highlight" is a better word than "point out" anyway, thank you